swagger/swagger.go

```diff
@@ -85,16 +91,20 @@ func New(config ...Config) fiber.Handler {
 		}
 	}
 
-	// Verify Swagger file exists
-	if _, err := os.Stat(cfg.FilePath); os.IsNotExist(err) {
-		panic(fmt.Errorf("%s file does not exist", cfg.FilePath))
-	}
+	rawSpec := cfg.FileContent
+	if len(rawSpec) == 0 {
+		// Verify Swagger file exists
+		_, err := os.Stat(cfg.FilePath)
+		if os.IsNotExist(err) {
```

Comment:
Use `errors.Is(err, fs.ErrNotExist)` instead

Reply:
That is existing code. I tried to minimize the changes. Do you still want me to change it?